### ISSUE TITLE
[REPLICATE_ATTRIBUTES] should check if tracked value has changed before updating

### ISSUE THREAD

**Issue by hariprasadiit** [anishkny/integrify#51]
In the code whether the tracked attributes value has changed or not is not checked. This results in writes even if there's no change to tracked attributes. 

```js
// Check if atleast one of the attributes to be replicated was changed
      let relevantUpdate = false;
      Object.keys(newValue).forEach(changedAttribute => {
        if (trackedMasterAttributes[changedAttribute]) {
          relevantUpdate = true;
        }
      });
```

We should be comparing old values of tracked attributes to new values, like

```js
// Check if atleast one of the attributes to be replicated was changed
      let relevantUpdate = false;
      Object.keys(newValue).forEach(changedAttribute => {
        if (trackedMasterAttributes[changedAttribute] && newValue[changedAttribute] !== oldValue[changedAttribute]) {
          relevantUpdate = true;
        }
      });
```

**Comment by anishkny** [comment]
Oh thats a great optimization @hariprasadiit! Although, did you mean:

```javascript
if (
  trackedMasterAttributes[changedAttribute] &&
  (newValue[changedAttribute] !== oldValue[changedAttribute])
) {
  relevantUpdate = true;
}
```

i.e. its relevant only if old and new values are **not** equal.

**Comment by hariprasadiit** [comment]
yes, the values should not be equal. typo!!

**Comment by anishkny** [comment]
Fixed by #59
